ha/setup/create_pacemaker_resources.py

```diff
@@ -302,14 +316,71 @@ def uds(cib_xml, push=False, **kwargs):
             op monitor timeout=30s interval=30s \
             op stop timeout=60s interval=0s")
 
-        # Constraints
-        process.run_cmd(f"pcs -f {cib_xml} pcs -f {cib_xml} constraint colocation add uds with csm-agent score=INFINITY")
-        process.run_cmd(f"pcs -f {cib_xml} pcs -f {cib_xml} constraint order csm-agent then uds")
 
-        if push:
-            cib_push(cib_xml)
+def config_constraint(cib_xml, push=False, **kwargs):
```

Comment:
I guess this method exists already, why are we adding new one?

Reply:
change was came through rebase, so removed it.